data/json/mutations.json

```diff
@@ -1423,6 +1423,76 @@
         "armor" : [ { "parts" : "ALL", "bash" : 1 } ]
     },{
         "type" : "mutation",
+        "id" : "LAMIA_SCALES_AMORED",
+        "name" : "Armored Scales",
+        "points" : 10,
+        "visibility" : 15,
+        "ugliness" : 4,
+        "description" : "Your tail and mouth have grown bullet resistant scales",
+        "cancels" : ["SCALES", "FEATHERS", "CHITIN", "PLANTSKIN", "FELINE_FUR", "LUPINE_FUR", "CHITIN_FUR"],
+        "prereqs" : ["LAMIA_TAIL"],
+        "threshreq" : ["THRESH_LAMIA"],
+        "category" : ["MUTCAT_LAMIA"],
+        "armor" : [ 
+            { "parts" : "LEG_L", "bash" : 11, "cut" : 24 }, 
+            { "parts" : "LEG_R", "bash" : 11, "cut" : 24 },
+            { "parts" : "FOOT_L", "bash" : 11 , "cut" : 24 },
+            { "parts" : "FOOT_R", "bash" : 11 , "cut" : 24 },
+            { "parts" : "MOUTH", "cut" : 14}
+            ]
```

Comment:
A minor acid protection (2-3 points) on the tail would be very useful, since you can't wear rubber boots.